cmd/viam-agent/main_linux.go

```diff
@@ -1,3 +1,6 @@
+//go:build linux
+// +build linux
+
```

Comment:
Nit: don't use these (rely on the filename itself) OR ALSO add them for the _windows file. I prefer the former, but as long as it's consistent.